src/components/VmActions/index.js

```diff
@@ -127,63 +127,59 @@ class VmActions extends React.Component {
       onForceShutdown,
       onSuspend,
       onRDP,
-      isOnCard,
       msg,
     } = this.props
     const isPoolVm = !!vm.getIn(['pool', 'id'], false)
     const isPool = !!pool && !isPoolVm
     const status = vm.get('status')
     const onStart = (isPool ? onStartPool : onStartVm)
 
-    const vncConsole = vm.get('consoles').find(c => c.get('protocol') === 'vnc')
+    const vncConsole = vm.get('consoles').find(c => c.get('protocol') === VNC)
     const hasRdp = isWindows(vm.getIn(['os', 'type']))
+    const defaultConsole = vm.get('defaultConsole')
```

Comment:
Is the `defaultConsole` being stored with the VM this way because you intend to make it user configurable in future?

Otherwise, it seems unnecessary to keep the system `defaultConsole` stored in the VM when it can just be accessed in `config`.